elastic/kibana
# [Security Solution] Fix toast message

## Summary

** Problem:
The toast message for installing templates / rules were hard coded, it always showed `Installed pre-packaged rules and timelines from elastic`

** This PR is to update the toast message, so it shows success message according to templates or rules users are installing.

Steps to verify:
- Installs only rules:
1. Go to timelines page, and make sure templates are loaded
2. Go to detection engine page and remove some prebuilt rules.
3. Click on the install button, and it should show `Installed pre-packaged rules from elastic` on the success toast.

- Installs only templates:
1. Go to detection engine page and make sure you have all the prebuilt rules installed.
2. Go to `kibana/x-pack/plugins/security_solution/server/lib/detection_engine/scripts` and run `sh ./timelines/delete_all_timelines.sh` to delete all the templates you have.
3. Click on the install button, and it should show `Installed pre-packaged timeline templates from elastic` on the success toast.


- Installs rules and templates:
1. Go to detection engine page and delete one or two prebuilt rules. 
2. Go to `kibana/x-pack/plugins/security_solution/server/lib/detection_engine/scripts` and run `sh ./timelines/delete_all_timelines.sh` to delete all the templates you have.
3. Click on the install button, and it should show `Installed pre-packaged rules and timeline templates from elastic` on the success toast.


![toast-templates](https://user-images.githubusercontent.com/6295984/97232179-9bd70e00-17d4-11eb-9dfa-9d0c73d74ad3.gif)

![toast](https://user-images.githubusercontent.com/6295984/97232245-b3ae9200-17d4-11eb-95d9-dc2283ce68b6.gif)


### Checklist

Delete any items that are not applicable to this PR.
